src/remoteConfig.ts

```diff
@@ -19,14 +20,14 @@ type RemoteConfigValue = {
     transfer: number;
   }>;
   approvedStoreKitSandboxUsers: string[];
-  coreProductId: string;
   coresRoleRules: {
     regions: string[];
     role: CoresRole;
   }[];
   kvasirRequirePlus: boolean;
   paddleIps: string[];
   paddleTestDiscountIds: string[];
+  paddleProductIds: Record<PurchaseType, string>;
```

Comment:
```suggestion
  paddleProductIds: Partial<Record<PurchaseType, string>>
```

so it denotes remote config is always optional